nsxt/resource_nsxt_policy_tier1_gateway_interface_test.go

```diff
@@ -231,9 +231,12 @@ func TestAccResourceNsxtPolicyTier1GatewayInterface_withIPv6(t *testing.T) {
 	ipv6Subnet := "4003::12/64"
 	updatedSubnet := fmt.Sprintf("%s\",\"%s", subnet, ipv6Subnet)
 	testResourceName := "nsxt_policy_tier1_gateway_interface.test"
-
+	profilePath := "/infra/ipv6-ndra-profiles/default"
+	if testAccIsGlobalManager() {
```

Comment:
pls consider using testAccAdjustPolicyInfraConfig for this replacement

Reply:
Will do. Thanks.